debian/syncd.install

```diff
@@ -1,3 +1,4 @@
 usr/bin/saidump
 usr/bin/saiplayer
 usr/bin/syncd*
+syncd/usr/bin/* usr/bin
```

Comment:
I was thinking of putting them in a `scripts` directory in the root, but since this directory contains the source of multiple Debian packages, I felt they were better off placed in the project directory. I can move them if you still think they belong under the root.